<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete</class>
 <widget class="QDialog" name="Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Game...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollAreaDeleteDialog">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>329</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelSavedGames">
         <property name="text">
          <string>Saved Games:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollAreaSavedGames">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="savedGamesArea">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>398</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="deleteButtonArea" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
